docs/src/overview/isolated-kubernetes.md

```diff
@@ -0,0 +1,401 @@
+|--------------------|----------------------|------------------------------------------------------|----------------------------------------------------------------------------------------------|
+| allow-to-http      | baseline             | 0.0.0.0/0                                            | egress via http                                                                              |
+| allow-to-https     | baseline             | 0.0.0.0/0                                            | egress via https                                                                             |
+| allow-to-apiserver | restricted, forbidden | IP of the Kubernetes API Server on the control plane | API Server communication of kubelet and other controllers                                    |
+| allow-to-dns       | all                  | IP of the private DNS Server                         | DNS resolution from the Kubernetes worker nodes and containers                               |
+| allow-to-ntp       | all                  | IP of the private NTP Server                         | Time synchronization                                                                         |
+| allow-to-registry  | restricted, forbidden | IP of the private Registry Mirror                    | Pulling strictly required container images                                                   |
+| allow-to-storage   | all                  | network of the container storage                     | persistent volumes with the cni driver                                                       |
+| allow-to-vpn       | all                  | IP of the vpn endpoint on the control plane          | allow communication from the api server to the kubelet for container logs and container exec |
+
+All of these CWNPs are managed by the gardener-extension-provider-metal, every manual modification will be reverted immediately.
+
+### Internet Access Forbidden
+
+This configuration can only be achieved by creating a new kubernetes cluster, it is not possible to modify a existing cluster (with internet access `baseline` or `restricted`) to this configuration. It is also required to specify the most recent version of kubernetes, older versions of kubernetes are not supported.
+
+Every network access modifications triggered by a cluster user, either by adding or modifying CWNPs or adding a Service Type Loadbalancer, is validated against the list of allowed Networks. If the cluster was created with a additional internal network, this network is part of the allowed networks list.
+
+With internet access `forbidden` the following restrictions apply:
+
+#### Egress traffic
+
+Egress traffic is only allowed to the private registry and the DNS and NTP servers. Additional CWNPs can be added to reach destinations in the internal networks if specified.
```

Comment:
Is it then possible to access an S3 endpoint

Reply:
Yes